moto/bedrock/models.py

```diff
@@ -0,0 +1,546 @@
+        vpc_config: Optional[Dict[str, Any]],
+    ) -> str:
+        if job_name in self.ModelCustomizationJobs.keys():
+            raise ResourceInUseException(
+                f"Model customization job {job_name} already exists"
+            )
+        if custom_model_name in self.CustomModels.keys():
+            raise ResourceInUseException(
+                f"Custom model {custom_model_name} already exists"
+            )
+        model_customization_job = ModelCustomizationJob(
+            job_name,
+            custom_model_name,
+            role_arn,
+            base_model_identifier,
+            training_data_config,
+            output_data_config,
+            hyper_parameters,
+            self.region_name,
+            self.account_id,
+            client_request_token,
+            customization_type,
+            custom_model_kms_key_id,
+            job_tags,
+            custom_model_tags,
+            validation_data_config,
+            vpc_config,
+        )
+        self.ModelCustomizationJobs[job_name] = model_customization_job
+        if job_tags:
+            self.tag_resource(model_customization_job.job_arn, job_tags)
+        # Create associated custom model
+        custom_model = CustomModel(
+            custom_model_name,
+            job_name,
+            model_customization_job.job_arn,
+            model_customization_job.base_model_arn,
+            model_customization_job.hyper_parameters,
+            model_customization_job.output_data_config,
+            model_customization_job.training_data_config,
+            model_customization_job.training_metrics,
+            model_customization_job.base_model_identifier,
+            self.region_name,
+            self.account_id,
+            model_customization_job.customization_type,
+            model_customization_job.output_model_kms_key_arn,
+            model_customization_job.validation_data_config,
+            model_customization_job.validation_metrics,
+        )
+        self.CustomModels[custom_model_name] = custom_model
+        if custom_model_tags:
+            self.tag_resource(custom_model.model_arn, custom_model_tags)
+        return model_customization_job.job_arn
+
+    def get_model_customization_job(self, job_identifier: str) -> ModelCustomizationJob:
+        if job_identifier not in self.ModelCustomizationJobs:
+            raise ResourceNotFoundException(
+                f"Model customization job {job_identifier} not found"
+            )
+        else:
+            return self.ModelCustomizationJobs[job_identifier]
+
+    def stop_model_customization_job(self, job_identifier: str) -> None:
+        if job_identifier in self.ModelCustomizationJobs:
+            self.ModelCustomizationJobs[job_identifier].status = "Stopped"
+        else:
+            raise ResourceNotFoundException(
+                f"Model customization job {job_identifier} not found"
+            )
+        return
+
+    def list_model_customization_jobs(
```

Comment:
Can we use the existing `@paginator`-decorator here? That way we can avoid all the custom pagination logic.

http://docs.getmoto.org/en/latest/docs/contributing/development_tips/utilities.html#paginator